libs/api/src/lib/graphql/blocks.ts

```diff
@@ -617,7 +661,7 @@ export const GraphQLCommentBlock = new GraphQLObjectType<CommentBlock, Context>(
       resolve: async ({filter}, _, {prisma}) =>
         prisma.comment.findMany({
           where: {
-            itemID: filter.item,
+            itemID: filter.item ?? '',
```

Comment:
Fixes issue of the comment block breaking articles/pages if done so before article or block exists.